.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Default Issue template
+about: Create an issue to help us improve
+title: ''
+labels: ''
```

Comment:
If you add a label, it will help manage them: `labels: triage` (without the simple quote, otherwise it won't work). Of course, the label `triage` has to be created.